site/docs/tutorial/java.md

```diff
@@ -47,6 +48,22 @@ In this tutorial you'll learn how to:
 To prepare for the tutorial, first [Install Bazel](../install.md) if
 you don't have it installed already.
 
+### Install the JDK
+
+1.  Install Java 8 JDK.
```

Comment:
I thought we are using a bundled JDK so users don't need to do this step?

Reply:
By my understanding, the bundled JDK is for just for Bazel itself to use. (I understand it will be significantly minimised in the future and stripped of things that Bazel doesn't need, so it would be impractical for Java developers.)

If I've understood these updated instructions correctly, this JDK is for Java developers themselves to use if they haven't already installed a JDK.

Reply:
Ah, I misunderstood it. Thanks.

Reply:
@jbduncan is right.